ddtrace/tracer/option.go

```diff
@@ -215,6 +218,14 @@ func WithDogstatsdAddress(addr string) StartOption {
 	}
 }
 
+// WithSamplingRules specifies the sampling rates to apply to spans based on the
+// provided rules.
+func WithSamplingRules(rules []*SamplingRule) StartOption {
```

Comment:
We should use pointers under the hood, but the API should not because it’s both uglier to write and we don’t want the user to hold a reference to an internally used structure.